### PR TITLE
Refactor strip_web_protocol()

### PR DESCRIPTION
**Pull request by JinguBangWest**
This allows for more prefixes to be added in the future if needed without repeating code multiple times.

### CODE REVIEW

File: tagstudio/src/core/utils/web.py [hunk 1]
```diff
@@ -4,9 +4,8 @@
...
-    new_str = string
-    new_str = new_str.removeprefix('https://')
-    new_str = new_str.removeprefix('http://')
-    new_str = new_str.removeprefix('www.')
-    new_str = new_str.removeprefix('www2.')
-    return new_str
+    prefixes = ['https://','http://','www.','www2.']
+    for prefix in prefixes:
+        if string.startswith(prefix):
+            string = string.replace(prefix, '')
```
Comment (yedpodtrzitko) [inline tagstudio/src/core/utils/web.py:10]: consider following example:

    >>> 'www.foowww.com'.replace('www.', '')
    'foocom'

Comment (Al3bad) [reply tagstudio/src/core/utils/web.py:10]: This should handle it. Basically replacing only the first occurance of the prefix

```
>>> 'www.foowww.com'.replace('www.', '', 1)
'foowww.com'
```

Comment (yedpodtrzitko) [reply tagstudio/src/core/utils/web.py:10]: ...why not keep it simple?

```diff
-       if string.startswith(prefix):
-            string = string.replace(prefix, '')
+       string = string.removeprefix(prefix)
```

Comment (JinguBangWest) [reply tagstudio/src/core/utils/web.py]: I meant to leave it as `string.removeprefix()` but didn't change it back for my commit. Also @yedpodtrzitko your right we should keep it simple. I've went ahead and changed this in my latest commit.

**Comment by Loran425** [comment]
Functionally identical, looks good.

**Pull request merged**